.gitignore

```diff
@@ -0,0 +1,58 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# Typescript v1 declaration files
+typings/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
```

Comment:
What do you think about `sudobot-*.tgz` so all tarballs aren't ignored?

Reply:
Are there tarballs you have in mind that we'd like to have in git, or would `sudobot-*.tgz` just be more faithful to the comment?

I just copied this from github/gitignore/master/Node.gitignore, so if it should be tailored more to our project, I'm fine with that. I just want to understand the reason for the change better.

Reply:
I have no tarballs in mind; just thinking defensively. Someone will surely one day have a tarball in this directory and maybe forget to add it to git because it doesn't get listed as untracked. :)